tests/test_extractcolumns.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+from bio_hansel.scripts.extract_test_columns import extract_test_columns
+from pandas.util.testing import assert_frame_equal
```

Comment:
👍 Nice use of an existing function! I'll have to do more digging into Pandas to see what they have to make testing easier!